docs/pages/application-access/okta/hosted-guide.mdx

```diff
@@ -90,6 +90,69 @@ Once the initial enrollment has completed, and if Teleport Identity
 is enabled, the enrollment workflow will guide you through enabling SCIM
 provisioning as well.
 
+## Configure Access List Synchronization
+
+If Teleport Identity is enabled, you will show you a guide to configure importing
+User Groups and Direct Application Assignments as access lists. This will ensure
+that permissions in Okta are modeled properly in Teleport, allowing Teleport users
+to see the same Okta applications that they would when viewing the Okta dashboard.
+
+### Configuring Default Owners
+
+![Set Default Owners](../../../img/enterprise/plugins/okta/okta-access-list-default-owners.png)
+
+Access lists must have owners, so the first step of setting up Okta Access List
+Synchronization is setting default owners for our new access lists. You can select
+any number of default owners and free-form enter the owners if they do not yet exist
+in the system. These owners can be later changed and will not be overwritten by the
+Okta Access List synchronization process.
+
+### Importing Okta User Groups as Access Lists
+
+![Import Okta User Groups](../../../img/enterprise/plugins/okta/okta-access-list-import-user-groups-all.png)
+
+After configurating default owners, you can customize which Okta User Groups should
+be imported. By default, all user groups will be imported. If you would like more
+control, you can uncheck the "Import All User Groups" checkbox at the top right of
+of the panel and configure multiple filters which will be matched against the group
+names.  For more information on filter syntax, please refer to the Import Filters section
```

Comment:
can we hyperlink/anchor the `Import Filters` section, also would renaming to `Using Filters` more accurate?

```suggestion
names.  For more information on filter syntax, refer to the Import Filters section
```